executor/runcexecutor/executor_linux.go

```diff
@@ -24,42 +22,40 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-func (w *runcExecutor) run(ctx context.Context, id, bundle string, process executor.ProcessInfo) (int, error) {
-	return w.callWithIO(ctx, id, bundle, process, func(ctx context.Context, pidfile string, io runc.IO) (int, error) {
-		return w.runc.Run(ctx, id, bundle, &runc.CreateOpts{
+func updateRuntime(runtime *runc.Runc) {
```

Comment:
could we use a more descriptive name for the functionality

Reply:
yeah, I didn't know what to call that.  I have updated to `updateRuncFieldsForHostOS` which is not great either but is more descriptive.  Let me know if you have a better name to use.